### ISSUE TITLE
'Show Balance' is not working with current version

### ISSUE THREAD

**Issue by The-Euphoria** [IncoCode/AccountInBank-GTA5#35]
Hi Inco, I wanted to thank you for making this mod, it's a really nice addition that really should have been in the vanilla game.

Anyway, the bugs are the same as other people have reported on the gta5-mods.com thread. With the latest version of GTA and all latest versions of your mod, Scripthook and NativeUI, you cannot see your balance, when you click 'show balance' it just freezes and does nothing.
Also, not every time, but a Lot of the time, when accessing an ATM, as soon as I click anything the game freezes then crashes. It seems multiple people have these issues :disappointed: 

I really hope you are still updating this mod, as it's the only one like it.
Anyway, Thanks again.

**Comment by IncoCode** [comment]
thanks for report, i will look into this issue this weekend, didn't check gta5-mods for a while.

**Comment by IncoCode** [comment]
just tryed and i cannot reproduce this issue. can you give me more details like your `ScriptHookVDotNet.log`, `AccountInBank.ini` and what mods are installed?

**Comment by The-Euphoria** [comment]
Hi, ok that's weird because I'm one of a few people having this issue :confused: 
- GTAV version is the latest update released just over a week ago (Can't remember the version number)
- Scripthook v1.0.678.1
- Scripthook.Net (Latest version)
- LUA Plugin for Script Hook V (Latest version)
- TrainerV (Latest version)

**MODS:**
- GTA Realism (Latest version)
- Realistic Top Speed v3
- PremiumDeluxeMotorsportCarDealership v3
- CarDealer Sell Cars (Latest version)
- And a lot of Car/Motorbike Replacement skins

That's all I had installed when I encounter the 'View Balance' issue. I'll upload the ScriptHookVDotNet.log, AccountInBank.ini in about an hour as I'm exporting a massive video right now so I can't run GTA at the moment.

**Comment by The-Euphoria** [comment]
Here you go, the ScriptHookVDotNet.log, AccountInBank.ini. I've changed them to .txt files just so they attach to this post.
Also, I've tried 4 or 5 different settings in the AIB.ini file, and I've tried without the mods I currently have installed.

Hope this helps.

[AccountInBankS - Copy.txt](https://github.com/IncoCode/AccountInBank-GTA5/files/276694/AccountInBankS.-.Copy.txt)
[ScriptHookVDotNet-2016-05-23 - Copy.txt](https://github.com/IncoCode/AccountInBank-GTA5/files/276695/ScriptHookVDotNet-2016-05-23.-.Copy.txt)

**Comment by IncoCode** [comment]
thanks. but can you please reproduce this error and send me the logs after that? because currently the logs says nothing.

**Comment by The-Euphoria** [comment]
That's the thing, I did reproduce it. I tried at 3 different ATM's, 4-5 times at each one. I tried putting money in first, transferring, leaving and coming back, I even clicked on View Balance then went off and made a drink in case it took a long time to load.... but nothing happened.
I can keep trying at an ATM until the game crashes if that will help, but it's pretty random, I was trying for over an hour last night with no crashes, but last week it crashed every time I went to an ATM.

**Comment by The-Euphoria** [comment]
Oh I almost forgot. I found an ATM last night at a gas station that wouldn't let me use it, I could press E to look at the normal vanilla account, but I couldn't look at your mods account. Even pressing B to mark the nearest ATM just mark one a few streets away.
I forgot to do it last night, but I'll screenshot where it is on the map for you later when I get a few minutes.

**Comment by The-Euphoria** [comment]
**[Update] --** IncoCode,
 I am extremely sorry, I've found the cause of the issue. It was a rogue .dll file left in the GTA folder. I purposely remove all mods when I test an issue so rule out any incompatibilities, but when I had an issue with the balance in your mod, I removed all my mod... '_But_' accidentally left 1 single .dll file in the GTA folder, which was causing the issue, and causing an issue with another mod I was trying to use.

And the very strange thing is, the mod is a 'Concussion mod', for realism when you take a baseball bat to the head or a 150mph head-on crash, is gives you blurry and swaying vision. Some how that was affecting your mod, but only one small part of it :confused: 
Anyway, I don't know the if this is the cause of the issue for the other people having this problem with Show Balance not working, but everything is now working for me.

Again, sorry to post a bug report and ask for help, it was my fault, I totally missed one of the files from a mod, it must have been in the folder for weeks because I removed that mod ages ago :disappointed: 
Thank you for your time, and Thank you sincerely for an absolutely awesome mod, a mod that really should have been in the vanilla game as it just makes sense.
